src/current/v24.1/molt-fetch.md

```diff
@@ -99,6 +103,12 @@ Complete the following items before using MOLT Fetch:
 
 - If a MySQL database is set as a [source](#source-and-target-databases), the [`--table-concurrency`](#global-flags) and [`--export-concurrency`](#global-flags) flags **cannot** be set above `1`. If these values are changed, MOLT Fetch returns an error. This guarantees consistency when moving data from MySQL, due to MySQL limitations. MySQL data is migrated to CockroachDB one table and shard at a time, using [`WITH CONSISTENT SNAPSHOT`](https://dev.mysql.com/doc/refman/8.0/en/commit.html) transactions.
 
+- To prevent memory outages during data export of tables with large rows, estimate the amount of memory used to export a table: `--row-batch-size * --export-concurrency * average size of the table rows`. If you are exporting more than one table at a time (i.e., [`--table-concurrency`](#global-flags) is set higher than `1`), add the estimated memory usage for the tables with the largest row sizes. Ensure that you have sufficient memory to run `molt fetch`, and adjust `--row-batch-size` accordingly.
+
+- If a table in the source database is much larger than the other tables, [filter and export the largest table](#schema-and-table-selection) in its own `molt fetch` task. Then export the remaining tables in another task.
+
+- When using [`IMPORT INTO` mode](#fetch-modes) to load tables into CockroachDB, if the fetch process terminates before the import job completes, you need to manually resume or cancel the job before running `molt fetch` again. Refer to [View and control import jobs]({% link {{ page.version.version }}/import-into.md %}#view-and-control-import-jobs).
```

Comment:
@ZhouXing19 I could use your guidance on how you re-run `molt fetch` after resuming/cancelling the job.

Reply:
```suggestion
- When using [`IMPORT INTO` mode](#fetch-modes) to load tables into CockroachDB, if the fetch process terminates before the import job completes, the hanging import job on the target database will keep the table offline. To bring this table accessible again for the next `molt fetch` attempt, you need to manually resume or cancel the job. Refer to [View and control import jobs]({% link {{ page.version.version }}/import-into.md %}#view-and-control-import-jobs).
```

Reply:
@ZhouXing19 Thanks! My question is whether the "next `molt fetch` attempt" requires running with continuation or just reissuing the same command and starting the import over from the beginning.

Reply:
Either case, they will need to bring the table online again by resuming or canceling the import job. Otherwise, the table just cannot be written into.

Reply:
I understand. But do you need to issue the continuation token to resume fetch from that table?

Reply:
Oh yes, if you want to resume you need the continuation token. You can also choose to restart from the very begining (i.e. redo export) without providing the token.

Reply:
@ZhouXing19 Got it -- updated!